db/seeds.rb

```diff
@@ -1,7 +1,41 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Author.destroy_all
+Author.create!([{
+  full_name: "Susan Cooper",
+  born: "in Burnham, Buckinghamshire, The United Kingdom ",
+  picture: "https://d.gr-assets.com/authors/1379606336p5/7308.jpg",
+  bio: "Susan Cooper was born in 1935, and grew up in England's Buckinghamshire, an area that was green countryside then but has since become part of Greater London. As a child, she loved to read, as did her younger brother,who also became a writer. After attending Oxford, where she became the first woman to ever edit that university's newspaper, Cooper worked as a reporter and feature writer for London's Sunday Times; her first boss was James Bond creator Ian Fleming.",
+  gender: "female"
+}, {
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.